web/ui/templates/status.html

```diff
@@ -54,7 +54,8 @@ <h2 id="alertmanagers">Alertmanagers</h2>
         </tr>
         {{range .Alertmanagers}}
         <tr>
-          <td>{{.}}</td>
+          {{/* Alertmanager URLs always have Scheme, Host and Path set */}}
+          <td>{{.Scheme}}://<a href="http://{{.Host}}">{{.Host}}</a>{{.Path}}</td>
```

Comment:
Okay, maybe I'm just being too clever then. I'll use `{{.Scheme}}` in the link. The important part is not linking to the `/api` URL.